render/wgpu/src/context3d/mod.rs

```diff
@@ -405,7 +405,6 @@ pub struct ShaderCompileData {
 #[collect(require_static)]
 pub struct TextureWrapper {
```

Comment:
Doesn't this mean that `TextureWrapper` can be removed altogether? Since now it's totally equivalent to `wgpu::Texture`.

Reply:
This requires a little more work - the impl needs to go in the base `ruffle_render` crate behind a feature, since the orphan rule requires us to wrote `impl Texture for wgpu::Texture` in the crate where the `Texture` trait is defined.

I'm working a refactor of this as part of my pixelbender work, and I'll include that change in the future PR.